package.json

```diff
@@ -2639,6 +2652,11 @@
                 "title": "Show Release Notes",
                 "category": "Brightscript"
             },
+            {
+                "command": "extension.brightscript.createNewProject",
```

Comment:
Let's remove the Roku: prefix here, it already shows up as "BrightScript: ` in the command palette, I think that's enough.